web_src/js/components/DiffFileTreeItem.vue

```diff
@@ -1,15 +1,19 @@
 <script lang="ts" setup>
 import {SvgIcon, type SvgName} from '../svg.ts';
 import {diffTreeStore} from '../modules/stores.ts';
-import {ref} from 'vue';
+import {computed, ref} from 'vue';
+import {fileIsViewed} from '../utils/filetree.ts';
 import type {Item, File, FileStatus} from '../utils/filetree.ts';
 
-defineProps<{
+const props = defineProps<{
   item: Item,
 }>();
 
 const store = diffTreeStore();
-const collapsed = ref(false);
+const isViewed = computed(() => {
+  return fileIsViewed(props.item);
+});
+const collapsed = ref(isViewed.value);
```

Comment:
`isViewed` is already a reactive value, why use `collapsed` to ref it again?

Reply:
Collapse folders marked as "read" by default on initial page load, while preserving manual expand/collapse toggle via clicks.

Reply:
The `collapsed` state controls whether a folder is folded, while `isViewed` indicates if a file/folder has been viewed. These two states can coexist:

- **Default Value Relation**: `collapsed` can initialize its default value from `isViewed` (as explained earlier), which is their only point of interaction.

- **​Independence in Logic**: Beyond initialization, their states operate independently in subsequent workflows.

Reply:
Okay, comments added.